README.md

```diff
@@ -35,20 +35,20 @@
 A (beta) modern embedded database. Doesn't your data deserve a (beta) beautiful new home?
 
 ```rust
-let tree = sled::open(path)?;
+let tree = sled::open("/tmp/welcome-to-sled").expect("open");
```

Comment:
The choice of /tmp here isn't very important.  I can even leave the path out entirely.  But I think adding the `.expect` is helpful since it doesn't immediately push the user into a function definition.